typescript/sdk/src/token/types.ts

```diff
@@ -101,8 +101,14 @@ export const HypTokenRouterConfigSchema = HypTokenConfigSchema.and(
 );
 export type HypTokenRouterConfig = z.infer<typeof HypTokenRouterConfigSchema>;
 
+const HypTokenRouterConfigMailboxOptionalSchema = HypTokenConfigSchema.and(
+  GasRouterConfigSchema.extend({
+    mailbox: z.string().optional(),
+  }),
+);
+
 export const WarpRouteDeployConfigSchema = z
-  .record(HypTokenRouterConfigSchema)
+  .record(HypTokenRouterConfigMailboxOptionalSchema)
```

Comment:
isn't the issue asking to remove the mailbox? this makes it optional. perhaps need clarification @nambrot

Reply:
Yeah I really mean optional, can be more precise going forward